task_schemas.md

```diff
@@ -241,7 +241,7 @@ Passages capture document structure, such as the title and abstact sections of a
 
 ## Textual Entailment
 
-- [Schema Template](nusantara/utils/schemas/entailment.py)
+- [Schema Template](nusacrowd/utils/schemas/entailment.py)
```

Comment:
Since `utils/schemas/entailment.py` is not exist, I think you can just remove this line

```suggestion
```